src/qibolab/instruments/zhinst.py

```diff
@@ -847,7 +847,6 @@ def drive(self, exp, qubits):
             if len(self.sequence[f"drive{q}"]) != 0:
                 play_after = None
                 for j, sequence in enumerate(self.sub_sequences[f"drive{q}"]):
-                    time = 0
```

Comment:
Won't this mess with https://github.com/qiboteam/qibolab/blob/89661e18c9a054eb76a4bde57a98e61756794f8d/src/qibolab/instruments/zhinst.py#L853C1-L856C34

Reply:
No, I don't think it will. The `drive_{i}` sequence starts after `drive_{i-1}`, therefore resetting the time to 0 would add an extra delay (of `pulse.pulse.start`, where `pulse` is the first pulse of `drive_{i}`) after the `drive_{i-1}` sequence.

Reply:
Okay, but in the code you are still doing `-time` at that line. Would it be undefined and complain ?

Reply:
No, it's defined here: https://github.com/qiboteam/qibolab/blob/89661e18c9a054eb76a4bde57a98e61756794f8d/src/qibolab/instruments/zhinst.py#L845

Reply:
You are right, I don't  know how to look at single commits properly, https://github.com/qiboteam/qibolab/pull/685/commits/89661e18c9a054eb76a4bde57a98e61756794f8d, and I got confused